lib/controller.dart

```diff
@@ -116,6 +123,24 @@ Map<String, Set> computeUniqFieldValues(List<model.Filter> filterOptions,
   return uniqueFieldValues;
 }
 
+// Render methods
+void handleNavToAnalysis() {
+  view.clearContentTab();
+  var tabLabels = _config.tabs
+      .asMap()
+      .map((i, t) => MapEntry(i, t.label ?? 'Tab ${i}'))
```

Comment:
It feels to me like you need to do this to account for one or more of the config.tabs potentially not having the label field set? Would it make sense instead to impose that the label is mandatory for a valid config, and do away with trying to generate one, or do you think that that would create too much inconvenience for the user?

Reply:
I think it is not too much to ask to name a tab. I will just make it mandatory - shouldn't be too much of an inconvenience for the user